tests/test_colic.py

```diff
@@ -44,8 +44,8 @@
                            TestCaseAnalyzer)
 
 
-NOMOS_PATH = "/home/slimbook/Escritorio/sources/graal-libs/nomossa"
-SCANCODE_PATH = "/home/slimbook/Escritorio/sources/graal-libs/scancode-toolkit-2.9.2/scancode"
+NOMOS_PATH = "/home/travis/build/chaoss/grimoirelab-graal/exec/fossology/src/nomos/agent/nomossa"
```

Comment:
I would move the two paths to a file in the tests folder (for instance `utils.py`) and then import the variables where needed. This would make easier to run the tests locally